test/integration/output/Relations.js

```diff
@@ -3558,7 +3558,6 @@ module.exports = {
           id: 1,
           site_id: 1,
           name: 'Main Site Blog',
-          _pivot_id: 1,
```

Comment:
Yes, the `id` on the Interim model has absolutely no effect on the query, so I guess there's no use for this pivot.